### DISCUSSION TITLE
Simple question

### DISCUSSION THREAD

**Discussion by assybish** [Open-Shell/Open-Shell-Menu#1809] [General]
A simple question . Windows 11 update April 2023 broke Open shell , caused explorer.exe to crash every few minutes , use massive CPU, crash taskbar, crash display background etc .. It has taken me 8 months to undo the damage. I no longer have open shell on the laptop sadly. Has there been any progress in open shell since April 2023 that would allow me to reinstall it and not conflict with the abomination that is win 11 .. sadly I am stuck with as all my programs and data are on the drive .?

**Comment by ge0rdi** [comment]
I'm not aware of any such issues.
`Open-Shell` works for me on fully updated Win11 (22H2 in my case).

I don't see any report from you about this issue.
If there is Explorer crashing you can provide dumps from it obtained via this [guide](https://github.com/Open-Shell/Open-Shell-Menu/wiki/How-to-create-dump-on-Windows-Explorer-crash).
Then we may be able to see what is going on and if it is `Open-Shell` related.

If you use some other customization tools, you can try to uninstall those one-by-one to see if the issue disappears (for example ExplorerPatcher used to have some issues after Windows updates).

**Reply by assybish** [reply]
I didn't report as it was and is common knowledge here is 1 reference https://github.com/Open-Shell/Open-Shell-Menu/issues/1580
Windows 11 is now 23H2 ! 

Another https://answers.microsoft.com/en-us/windowserver/forum/all/windows-11-issue-after-22h2-update-the-shell/830e4cd5-245c-49e0-904a-ed8f3df291f9
https://www.elevenforum.com/t/windows-11-changes-its-mind-when-open-shell-is-installed.17979/
https://www.windowslatest.com/2023/04/13/windows-11-kb5025239-issues-file-explorer-ssd-bsod/
Finally msoft own mealy mouthed words https://www.windowslatest.com/2023/04/13/windows-11-kb5025239-issues-file-explorer-ssd-bsod/

**Reply by ge0rdi** [reply]
> I didn't report as it was and is common knowledge here is 1 reference https://github.com/Open-Shell/Open-Shell-Menu/issues/1580

That ticket doesn't mention any Explorer crashes as you mentioned.
I don't think it is anyhow related to your issues.

> Another ...

Don't see any reference to `Open-Shell` there ...
It mentions `ExplorerPatcher` that is known to have compatibility issues with new Windows builds (that's why I asked if you are using another such customization software).

> Finally msoft own mealy mouthed words

Again, no reference to `Open-Shell` there.
I guess there may have been some issue in some Windows 11 update, but not sure if that's related to `Open-Shell` anyhow.

**Reply by ge0rdi** [reply]
> Windows 11 is now 23H2 !

Installed `Open-Shell` `4.4.191` into fully updated Win11 23H2 and haven't noticed any issue.
